tests/unittests/MLTest.cpp

```diff
@@ -205,6 +207,79 @@ TEST_P(MLTest, learnXor) {
   }
 }
 
+/// Learn the logarithmic function
+TEST_P(MLTest, learnLog) {
+  unsigned numInputs = 500;
+  unsigned batchSize = 10;
+  EE_.getConfig().learningRate = 0.05;
+  EE_.getConfig().batchSize = batchSize;
+
+  auto &mod = EE_.getModule();
+  Function *F = mod.createFunction("main");
+  F->setName("learnLog");
```

Comment:
Why do you create the function with the name "main" just to change it a line after.

Reply:
Hi. I was following the convention of other test cases. I thought it was a required routine here. Do I just create the function with its name passed to it as a parameter?

Reply:
If the documents don't specify this then we should update the docs.